substrate/frame/distribution/src/functions.rs

```diff
@@ -0,0 +1,163 @@
+// This file is part of Substrate.
+
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Helper functions for Distribution pallet.
+
+pub use super::*;
+impl<T: Config> Pallet<T> {
+	pub fn pot_account() -> AccountIdOf<T> {
+		// Get Pot account
+		let pot_id = T::PotId::get();
+		let pot_account: AccountIdOf<T> = pot_id.into_account_truncating();
+		pot_account
+	}
+
+	pub fn get_spend(project_account: ProjectId<T>) -> Vec<SpendIndex> {
+		let mut spends: Vec<SpendIndex> = Vec::new();
+		let value = Some(project_account);
+
+		for spend in Spends::<T>::iter() {
```

Comment:
```suggestion
		for(index, info) in Spends::<T>::iter() {
```

Reply:
not super important, but you can rewarite this to `.iter().filter_map(...).collect()`